common/source/docs/common-nooploop.rst

```diff
@@ -0,0 +1,87 @@
+.. _common-nooploop:
+
+=======================================
+	Nooploop for Non-GPS Navigation
+=======================================
+
+[copywiki destination="copter,rover"]
+
+This article explains how a `Nooploop <https://www.nooploop.com/en/>`__ system based on UWB(Ultra Wide Band) technology can be used as a short-range substitute for a GPS allowing position control modes like Loiter, PosHold, RTL, Auto indoors.
+
+Required Hardware
+=================
+
+* 5 Nooploop LinkTrack nodes.
+* one `Arduino Nano Every <https://store.arduino.cc/usa/nano-every>`__ (optional)
+* 4 `portable chargers <https://www.amazon.com/s/ref=nb_sb_noss_2?url=search-alias%3Daps&field-keywords=portable+charger&rh=i%3Aaps%2Ck%3Aportable+charger>`__ with type A USB ports to power the anchors (optional)
+* 4 `tripods <https://www.amazon.com/s/ref=nb_sb_noss_1?url=search-alias%3Daps&field-keywords=tripod&rh=i%3Aaps%2Ck%3Atripod>`__ to mount the anchors (optional)
+
+Preparing the anchors
+=====================
+
+4 Nooploop nodes need to be configured to act as anchors. Follow Nooploop 'LinkTrack user manual <https://www.nooploop.com/en/download/>'__ to setup and calibate them. 
```

Comment:
need space after underline character at end of link for link to format correctly